tensorflow/python/data/experimental/ops/readers.py

```diff
@@ -341,6 +341,26 @@ def make_csv_dataset_v2(
   maps feature column names to `Tensor`s containing the corresponding
   feature data, and labels is a `Tensor` containing the batch's label data.
 
+  Usage Example:
+
+  Using IRIS dataset to show how to convert .csv file into a dataset.
+
+  >>> train_dataset_url = "https://storage.googleapis.com/download.tensorflow.org/data/iris_training.csv"
+  >>> train_dataset_fp = tf.keras.utils.get_file(fname=os.path.basename(train_dataset_url), origin=train_dataset_url)
```

Comment:
Can you rewrite this test so it doesn't hit the network on every test?

This causes issues on test environments with limited/no internet access

Reply:
Should I do something like 

`train_dataset_url = "path/to/file.csv"`

Reply:
Yes, and add the csv to `data` in the corresponding `BUILD` bazel target

Reply:
It will hit the network anyway, can i make it a usage example like @rachellim said ?

Reply:
Ok